### PR TITLE
docs/esp32/peripheral_access: Access to pins via registers.

### PR DESCRIPTION
**Pull request by IhorNehrutsa**
Synchronous access to pins directly via registers.
Pins 16 and 17 are switched synchronously:
image
Pins 32 and 33 are switched synchronously also.

### CODE REVIEW

**Comment by andrewleech** [comment]
This looks like a rather interesting example to add to the docs! 

Note the ci failed on commit message formatting due to it requiring a signed-off line in the commit message, there's some details about this here: https://github.com/micropython/micropython/blob/master/CODECONVENTIONS.md

**Comment by IhorNehrutsa** [comment]
@andrewleech 
Thanks.

Another question. How I can change the **Labels** field?
It is not available to me. It is gray.
image

**Comment by andrewleech** [comment]
> Another question. How I can change the **Labels** field?
> It is not available to me. It is gray.
> 

Ah yeah only maintainers can add labels, the privileges system in GitHub is weird.

**Comment by IhorNehrutsa** [comment]
Thanks @andrewleech @robert-hh

File: docs/esp32/tutorial/peripheral_access.rst [hunk 1]
```diff
@@ -42,3 +42,73 @@ for this:
 
 The MCPWM0 peripheral is in bit position 17 of the above two registers, hence
 the value of ``DPORT_PWM0_CLK_EN``.
+
+Synchronous access to pins directly via registers
+-------------------------------------------------
+
+.. code-block:: python3
```
Comment (dpgeorge) [inline docs/esp32/tutorial/peripheral_access.rst:49]: I suggest adding a short paragraph before the code example, to explain what it is doing.

Comment (IhorNehrutsa) [reply docs/esp32/tutorial/peripheral_access.rst:53]: Added

**Comment by dpgeorge** [comment]
Thanks for updating the PR.  I also tweaked the grammar a bit in the description paragraph.

**Comment by IhorNehrutsa** [comment]
Thanks

**Pull request merged**